src/RestControllers/RestControllerHelper.php

```diff
@@ -12,8 +12,37 @@
 
 namespace OpenEMR\RestControllers;
 
+use OpenEMR\Common\Logging\SystemLogger;
+use OpenEMR\Common\System\System;
+use OpenEMR\FHIR\FhirSearchParameterType;
+use OpenEMR\FHIR\R4\FHIRDomainResource\FHIROperationDefinition;
+use OpenEMR\FHIR\R4\FHIRDomainResource\FHIRPatient;
+use OpenEMR\FHIR\R4\FHIRElement\FHIRCanonical;
+use OpenEMR\FHIR\R4\FHIRElement\FHIRCode;
+use OpenEMR\FHIR\R4\FHIRElement\FHIRExtension;
+use OpenEMR\FHIR\R4\FHIRElement\FHIRReference;
+use OpenEMR\FHIR\R4\FHIRElement\FHIRRestfulCapabilityMode;
+use OpenEMR\FHIR\R4\FHIRElement\FHIRString;
+use OpenEMR\FHIR\R4\FHIRElement\FHIRTypeRestfulInteraction;
+use OpenEMR\FHIR\R4\FHIRResource;
+use OpenEMR\FHIR\R4\FHIRResource\FHIRCapabilityStatement\FHIRCapabilityStatementInteraction;
+use OpenEMR\FHIR\R4\FHIRResource\FHIRCapabilityStatement\FHIRCapabilityStatementOperation;
+use OpenEMR\FHIR\R4\FHIRResource\FHIRCapabilityStatement\FHIRCapabilityStatementResource;
+use OpenEMR\FHIR\R4\FHIRResource\FHIRCapabilityStatement\FHIRCapabilityStatementRest;
+use OpenEMR\Services\FHIR\IResourceUSCIGProfileService;
+
 class RestControllerHelper
 {
+    /**
+     * The resource endpoint names we want to skip over.
+     */
+    const IGNORE_ENDPOINT_RESOURCES = ['.well-known', 'metadata', 'auth'];
```

Comment:
is auth an endpoint anymore?

Reply:
I searched _rest_routes.inc.php and didn't see it, so I've gone ahead and removed it.